client/controller/index.web.js

```diff
@@ -40,6 +42,18 @@ export const ProviderWrappedLayout = ( {
 	const state = store.getState();
 	const userLoggedIn = isUserLoggedIn( state );
 
+	useEffect( () => {
```

Comment:
Changed the approach to load the experiment based on @candy02058912  [PR](https://github.com/Automattic/wp-calypso/pull/72945)

Reply:
I saw @tyxla [commented about this code](https://github.com/Automattic/wp-calypso/pull/72945/files#r1095820066) in @candy02058912's PR. Linking here to pull it into the loop.

Reply:
Thanks, @DustyReagan, added [the validation here](https://github.com/Automattic/wp-calypso/pull/72932/commits/8467502c2827e3d4a308f65763af209fe915a728). Now we should be good to go 🤞

Reply:
Hey @tyxla, can you have a glance at this code to ensure it's good to go?